# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openlibraries/compile.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- openlibraries-0.3.0/src/openobjectlib/sg/appearance.cpp_orig	2007-09-06 22:00:23.000000000 +0200
+++ openlibraries-0.3.0/src/openobjectlib/sg/appearance.cpp	2007-09-06 22:01:14.000000000 +0200
@@ -54,7 +54,8 @@
 	{
 		typedef opl::m_url::const_iterator const_iterator;
 		
-		const_iterator I = std::find_if( m.begin( ), m.end( ), fs::exists );
+		typedef bool (*fileSystemFunc)( const boost::filesystem::path & ph );
+		const_iterator I = std::find_if<opl::m_url::const_iterator, fileSystemFunc>(m.begin() , m.end(), fs::exists );
 		if( I != m.end( ) )
 		{
 			p = *I;
--- openlibraries-0.3.0/src/openmedialib/plugins/avformat/avformat_plugin.cpp_orig	2007-09-06 21:49:47.000000000 +0200
+++ openlibraries-0.3.0/src/openmedialib/plugins/avformat/avformat_plugin.cpp	2007-09-06 21:50:09.000000000 +0200
@@ -4,7 +4,7 @@
 // Copyright (C) 2005-2006 VM Inc.
 // Released under the LGPL.
 // For more information, see http://www.openlibraries.org.
-
+#define __STDC_CONSTANT_MACROS
 
 
 
